parking_permits/models/driving_class.py

```diff
@@ -15,12 +15,16 @@
     ),
     "B": (
         VehicleClass.M1,
-        VehicleClass.M2,  # M2 mass less than 3500KG
+        VehicleClass.M1G,
```

Comment:
Just to confirm: is there a migration needed for this additional choice?

Reply:
True, it is needed. Added it now.